src/components/Jobs/JobView/index.tsx

```diff
@@ -166,47 +145,81 @@ interface IJobApplicants {
   showJobApplicants: any;
   onClick: any;
   JobApplicantsData: any;
+  orgOwnerUserId: any;
+  setEditJobListing: any;
+  jobUuid: any;
 }
 
 const JobApplicants: FC<IJobApplicants> = ({
   setShowJobApplicants,
   showJobApplicants,
   onClick,
   JobApplicantsData,
+  orgOwnerUserId,
+  setEditJobListing,
+  jobUuid,
 }) => {
+  const userId = useSelector((state: RootState) => state.user.user?.userId);
+
+  const userIsOrgOwner = () => {
+    return orgOwnerUserId === userId;
+  };
+
+  const editJobEntry = () => {
+    setEditJobListing(true);
+  };
+
+  const generateShareLink = () => {
+    navigator.clipboard.writeText(`${config.AppDomain}/jobs/all?${jobUuid}`);
+    toast.success('Copied to clipboard!');
+  };
+
   return (
     <>
-      <div className={styles['job-list-people-wrap']} onClick={onClick}>
+      <div className={styles['job-list-people-wrap']}>
         <div className={styles['job-list-people']} key={getRandomString(5)}>
-          {JobApplicantsData && JobApplicantsData.length === 0 && (
-            <p>no applicants yet.</p>
-          )}
-          {JobApplicantsData &&
-            JobApplicantsData.slice(0, 3).map((item: any) => {
-              return (
-                <>
-                  <div
-                    className={styles['job-people']}
-                    key={getRandomString(5)}
-                  >
-                    <img
-                      src={
-                        item.Profile.picture ||
-                        getDefaultAvatarSrc(
-                          item?.Profile?.user?.name?.charAt(0).toUpperCase()
-                        )
-                      }
-                      alt=""
-                    />
-                  </div>
-                  {JobApplicantsData && JobApplicantsData.length > 2 && (
-                    <div>
-                      <p>+ {JobApplicantsData.length}</p>
-                    </div>
-                  )}
-                </>
-              );
-            })}
+          <span>
+            {JobApplicantsData && JobApplicantsData.length === 0 && (
```

Comment:
What's the difference in between this and the thing below this? Both of them are same buttons.

Reply:
Checking if there are no job applicants, in case there are none, first button is showed else second one.

Reply:
But both of them are same buttons